testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2906,6 +2907,53 @@ public void testCreateTable()
         assertFalse(getQueryRunner().tableExists(getSession(), tableNameLike));
     }
 
+    @Test
+    public void testShowTables()
```

Comment:
i.e. if i understand the purpose of the test it's to verify CREATE + SHOW shows the new object. A DROP + SHOW doesn't show the object. We do exact sequence in `testCreateTable` and `testView` with only difference being that instead of running `SHOW` we use some other mechanism.

Reply:
If this is not exact duplicate of an existing test, i'd prefer to keep this one.
we need explicit test coverage for SHOW TABLES, don't we?

Reply:
we have that already `testShowTables` and `testShowTablesLike`?

If anything I'd actually propose to modify that test instead?

Reply:
pushed an update, ptal